core/common/no_cb.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_COMMON_NO_CB_HPP
+#define KAGOME_COMMON_NO_CB_HPP
+
+#include <cstddef>
+
+namespace kagome {
+  /**
+   * Explicitly specify empty `std::function` parameter.
+   * All `std::function<F>` have `nullptr_t` constructor.
+   *
+   * Example:
+   *   void foo(std::function<void()> optional_cb);
+   *   foo(useful_cb);
+   *   foo(kNoCb);
+   */
+  constexpr auto kNoCb = nullptr;
```

Comment:
Not very readable to be honest, I think it's not worth it to shortify 'callback' here because when you just see it as an argument it can take a while to figure out what a 'knocb' means. I suggest k(No/Empty/Idle)(Callback/Func/Op).